src/Goerli-DssSpell.sol

```diff
@@ -47,6 +51,9 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
         // Includes changes from the DssSpellCollateralAction
         // onboardNewCollaterals();
         // offboardCollaterals();
+
+        StarknetLike(DssExecLib.getChangelogAddress("STARKNET_DAI_BRIDGE")).setMaxDeposit(1000 * WAD);
```

Comment:
note: we usually include a comment with action description and poll link, that in this particular case should look like as follows:
```suggestion
        // Increase Starknet Bridge Deposit Limit from 50 DAI to 1000 DAI
        // https://vote.makerdao.com/polling/QmbWkTvW
        StarknetLike(DssExecLib.getChangelogAddress("STARKNET_DAI_BRIDGE")).setMaxDeposit(1000 * WAD);
```